datalogistik/dataset.py

```diff
@@ -424,7 +452,10 @@ def validate(self):
         log.info(f"Dataset is{'' if dataset_valid else ' NOT'} valid")
         return dataset_valid
 
-    def get_write_format(self, table):
+    def get_write_format(self, table: Table):
```

Comment:
```suggestion
    def get_write_format(self, table: Table) -> Tuple[pads.FileFormat, Optional[dict]]:
```
...pretty sure this is wrong, sorry